src/main/java/kr/co/antoon/user/domain/vo/Status.java

```diff
@@ -0,0 +1,23 @@
+package kr.co.antoon.user.domain.vo;
+
+import kr.co.antoon.oauth.application.CustomOAuth2UserService;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum Status {
+
+    SUCCESS("로그인 성공"),
+    SIGNUP("회원가입"),
+    ;
+
+    private final String description;
+
+    public static Status of(Boolean check) {
+        if (!check) {
+            return SIGNUP;
+        }
+        return SUCCESS;
```

Comment:
부정으로 시작하는 것보다는 긍정으로!

```suggestion
        if (check) {
            return SUCCESS;
        }
        return SIGNUP;
```